Description: Do not use timestamp macros to allow reproducible builds.
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2016-04-09

--- a/M2/Macaulay2/bin/timestamp.c
+++ b/M2/Macaulay2/bin/timestamp.c
@@ -1 +1 @@
-char timestamp[] = __DATE__ ", " __TIME__;
+char timestamp[] = "omitted to allow reproducible builds";
